fetch.bs

```diff
@@ -5788,6 +6077,10 @@ due course.
 <dfn id=concept-body-mime-type for=Body>MIME type</dfn> algorithm which takes no arguments and
 returns failure or a <a for=/>MIME type</a>.
 
+<p>Objects including the {{Body}} interface mixin have an associated
+<dfn id=concept-body-on-fully-read for=Body>on body consumed</dfn> (an algorithm which takes no
```

Comment:
Let's call this fully read, like the other one. No need for ID. (Or is Bikeshed case-insensitive? In that case, maybe "once read". I'd rather not name it similarly to event handlers.)